app/views/idv/doc_auth/_back.html.erb

```diff
@@ -0,0 +1,27 @@
+<%#
+locals:
+* action: (Optional) Flow action to call to return to the previous step.
+* step: (Optional) Name of step to which user should be returned.
+* path: (Optional) Path to which user should navigate.
+%>
+<%
+if [:action, :step, :path].all? { |key| !local_assigns[key] }
+  raise 'must pass one of action, step, or path'
+end
+
+text = '‹ ' + t('forms.buttons.back')
+step = local_assigns[:action] || local_assigns[:step]
+path = local_assigns[:path] || idv_doc_auth_step_path(step: step)
+%>
+<div class="margin-top-5 padding-top-2 border-top border-primary-light">
+  <% if local_assigns[:action] %>
+    <%= button_to(
+      text,
+      path,
+      method: :put,
+      class: 'btn btn-link'
+    ) %>
+  <% else %>
+    <%= link_to(text, path) %>
+  <% end %>
```

Comment:
There's an alternative here where we use `link_to` in combination with the `method` option to flatten this to a single logic path:

https://api.rubyonrails.org/v5.2.3/classes/ActionView/Helpers/UrlHelper.html#method-i-link_to

But this requires [unobtrusive scripts](https://github.com/rails/rails/tree/main/actionview/app/assets/javascripts), which we currently don't use, and is maybe a bit hefty for such purpose. That being said, it was a bit confusing for me to not be able to rely on documented options to work as-is, since they clearly assume UJS scripts exist 🤷